Source/Prism.Tests/Commands/DelegateCommandFixture.cs

```diff
@@ -374,7 +374,7 @@ public void NonGenericDelegateCommandObservingPropertyShouldRaiseOnEmptyProperty
 
             command.CanExecuteChanged += delegate { canExecuteChangedRaised = true; };
 
-            OnPropertyChanged(null);
+			RaisePropertyChanged();
```

Comment:
Using xamarin studio by any chance? Tabs vs spaces ;) (codebase is using spaces)